src/themes/elife/lib/downloads.ts

```diff
@@ -40,45 +42,17 @@ const buildMenu = (
 
       create('h2', null, 'Download links'),
       create('h3', null, 'Downloads'),
-      create(
-        'ul',
-        null,
-        create(
-          'li',
-          null,
-          create(
-            'a',
-            {
-              href: pdfUrl,
-              target: '_parent',
-              'data-is-download-pdf-link': true,
-            },
-            'Article PDF'
-          )
-        )
-      ),
```

Comment:
This has now returned the list item within the list item rather than them being separate

Reply:
This is addressed in this PR: https://github.com/stencila/thema/pull/200

Reply:
Ignore me. will resolve this